server/channels/api4/team_test.go

```diff
@@ -2298,8 +2322,8 @@ func TestAddTeamMember(t *testing.T) {
 	_, resp, err = client.AddTeamMemberFromInvite(context.Background(), token.Token, "")
 	require.Error(t, err)
 	CheckNotFoundStatus(t, resp)
-	th.App.DeleteToken(token)
-
+	appErr = th.App.DeleteToken(token)
+	require.Nil(t, appErr)
 	// by invite_id
```

Comment:
Nit: Let's add back the newline
```suggestion
	require.Nil(t, appErr)
	// by invite_id

```